### PR TITLE
chore: import stub `Vagrantfile`

### PR DESCRIPTION
**Pull request by NicolasT**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
```
Comment (slaperche-scality) [inline Vagrantfile]: minor: could add `# frozen_string_literal: true` to be compatible with future Ruby versions.

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+$IMPORT_RELEASE = <<-SCRIPT
```
Comment (slaperche-scality) [inline Vagrantfile]: Do we need a global variable here? A constant would be enough, no?

File: Vagrantfile [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+# This script is supposed to 'install' the build results on the system,
+# i.e. symlink something like `/vagrant/_build/root` to
+# `/srv/scality/metalk8s-$VERSION` or however we build this.
+
+echo "Installed build artifacts on the host"
+
+exit $RC
+SCRIPT
+
+$BOOTSTRAP =<<-SCRIPT
```
Comment (slaperche-scality) [inline Vagrantfile]: missing space after `=`

File: Vagrantfile (hunk 1 above)
Comment (NicolasT) [reply Vagrantfile]: This is not really a full-fledged Ruby script, but sure, will do. Also, I'm not a Ruby dev, this file is mostly knit together from stuff found in the Vagrant docs :innocent:

**Comment by NicolasT** [comment]
@bert-e approve

**Review by slaperche-scality — APPROVED** [review]

**Pull request merged**